core-bundle/contao/templates/twig/backend/data_container/breadcrumb.html.twig

```diff
@@ -0,0 +1 @@
+{{ knp_menu_render('be_breadcrumb_menu', {allow_safe_labels: true, template: '@ContaoCore/Menu/backend_breadcrumb.html.twig'}) }}
```

Comment:
We do not use Symfony templates in Contao (anymore) and we must not add new ones - I just moved the error templates recently, which were the last remainders. 😉

This should instead be something like `@Contao/backend/menu/_breadcrumb.html.twig`.